File: task-03/Java/Subtask4.java
import java.nio.file.*;

public class Main {
    public static void main(String[] args) {
        // Read the number from input.txt
        String data = Files.readString(Path.of("input.txt")).trim();
        int n = Integer.parseInt(data);

        // Create diamond pattern
        StringBuilder diamond = new StringBuilder();

        // Top half
        for (int i = 0; i < n; i++) {
            diamond.append(" ".repeat(n - i - 1));
            diamond.append("*".repeat(2 * i + 1));
            diamond.append("\n");
        }

        // Bottom half
        for (int i = n - 2; i >= 0; i--) {
            diamond.append(" ".repeat(n - i - 1));
            diamond.append("*".repeat(2 * i + 1));
            diamond.append("\n");
        }

        // Write diamond pattern to output.txt
        Files.writeString(Path.of("output.txt"), diamond.toString());
    }
}

File: task-03/Javascript/Subtask4.js
const fs = require('fs');

// Read the number from input.txt
const n = parseInt(fs.readFileSync('input.txt', 'utf8'));

// Create the diamond pattern
let diamond = '';

// Top half of the diamond
for (let i = 0; i < n; i++) {
    diamond += ' '.repeat(n - i - 1) + '*'.repeat(2 * i + 1) + '\n';
}

// Bottom half of the diamond
for (let i = n - 2; i >= 0; i--) {
    diamond += ' '.repeat(n - i - 1) + '*'.repeat(2 * i + 1) + '\n';
}

// Write the diamond pattern to output.txt
fs.writeFileSync('output.txt', diamond);

